src/javascript/app/ng-wig/ng-wig.component.js

```diff
@@ -51,7 +51,10 @@ angular.module('ngWig')
 
       this.$onInit = () => {
         let placeholder = Boolean(this.placeholder);
-        this.ngModelController.$render = () =>  !placeholder ? $container.html(this.ngModelController.$viewValue || '<p></p>') : $container.empty();
+        this.ngModelController.$render = () => this.ngModelController && this.ngModelController.$viewValue
```

Comment:
So `this.ngModelController` is not needed, only `this.ngModelController.$viewValue`. Right?